src/main/java/core/basesyntax/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    private static final String CSV_DELIMITER = ",";
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> rawData) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        if (rawData == null || rawData.isEmpty()) {
+            return transactions;
+        }
+
+        for (String line : rawData) {
+            if (line == null || line.trim().isEmpty() || line.startsWith("type,")) {
```

Comment:
```suggestion
            if (line == null || line.trim().isEmpty())) {
```
Don't skip such line, throw exception